### ISSUE TITLE
NTR: VZV present in swab from skin or mucous membranes (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8104]
New term request VZV present in swab from skin or mucous membranes (seidel_190602160532): Varicella zoster virus; PCR or viral culture; chickenpox; shingles; herpes zoster  UMLS: 2362670

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000905
- label: Positive cutaneous varicella zoster virus nucleic acid test
- parents: [Positive skin infectious agent test (HP:0430117)](https://hpo.jax.org/app/browse/term/HP:0430117)
- pmid: 
- def: Detection of mucocutaneous varicella zoster virus nucleic acid in the skin by a method such as polymerase chain reaction. The sample is usually obtained by swab from skin or mucous membranes.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8104
- synonyms:
